_blog/kubernetes-on-solar-plants/index.md

```diff
@@ -171,15 +159,15 @@ When you update your package and wish to redistribute it, you can only ship the
 
 {% include_relative layers.html %}
 
-When the delta is received, the new package is recomputed from the diff, unzipped and run as a separate process.
+When the delta is received, the new package is recomputed from the diff, unzipped and run as a separate process. <!-- this is cool - have you seen https://www.balena.io/os/docs/ ?-->
```

Comment:
Yes, I originally thought about using ResinOS. But it turns out that ResinOS is not as flexible as k8s. You have to follow their process to release software updates. Kubernetes, on the other hand, is not designed to run on embedded hardware, but it's completely customisable.